### PR TITLE
WIP: Add package

### PR DESCRIPTION
**Pull request by aureocarneiro**

### CODE REVIEW

**Comment by aureocarneiro** [comment]
Hej @AntoineDupre I need to test the package and dep. yet wait for me

**Comment by aureocarneiro** [comment]
Hey @AntoineDupre you can start the review!

I add some new features...

1st - I am using now OptionParser to add a --help option

```
from optparse import OptionParser

usage = "%prog [options] <pool_instance> "
parser = OptionParser(usage)
options, args = parser.parse_args()

if len(args) == 0:
    logging.fatal("You need to provide at least a pool instance")
    sys.exit()
```

2 - I am taking the file with a abs path

```
r_workbook = xlrd.open_workbook("{}/template/template.xls".format(os.path.dirname(os.path.realpath(__file__))))
```

3 - Now don't meter wherever I am, when I run the scrip the result xls file will be save in my path

```
w_workbook.save("{}/{}.xls".format(os.getcwd(), pool))
```

4 - I add .gitignore file to don't save the output (.xls file)  d62eed5

5 - I pass the template.xls with MANIFEST.in  4989e58

6 - I refectory the main code to use globals variables ad1aece

I will wait your review to publish on maxiv gitlab.
I will also create a .gitlabci file 
Thanks!

**Comment by aureocarneiro** [comment]
@AntoineDupre I installed on kitslab-cc-0, you can test there !

**Comment by aureocarneiro** [comment]
I need to update the README file also

File: sardana2xls/sardana2xls.py [hunk 1]
```diff
@@ -8,17 +8,27 @@
...
 from functools import partial
 import sys
+import os
 
 import logging
 
 # logging.basicConfig(level=logging.INFO)
 logging.basicConfig(level=logging.DEBUG)
 
+from optparse import OptionParser
```
Comment (AntoineDupre) [inline sardana2xls/sardana2xls.py]: maybe it is better to use `argparse` (optparse is deprecated)/

File: sardana2xls/sardana2xls.py [hunk 2]
```diff
@@ -64,7 +74,7 @@
 
 
 # Open xls file
-r_workbook = xlrd.open_workbook("template.xls")
+r_workbook = xlrd.open_workbook("{}/template/template.xls".format(os.path.dirname(os.path.realpath(__file__))))
```
Comment (AntoineDupre) [inline sardana2xls/sardana2xls.py]: It is maybe a two long line (112 chars)

```python
module_path = os.path.dirname(os.path.realpath(__file__))
template_path = "{}/template/template.xls".format(module_path)
r_workbook = xlrd.open_workbook(template_path)
```

**Review by AntoineDupre — COMMENTED** [review]
Looks good,
Thanks !

File: sardana2xls/sardana2xls.py (hunk 1 above)
Comment (aureocarneiro) [reply sardana2xls/sardana2xls.py]: Its true ! Nice!

File: sardana2xls/sardana2xls.py (hunk 2 above)
Comment (aureocarneiro) [reply sardana2xls/sardana2xls.py]: Good point!

Comment (aureocarneiro) [reply sardana2xls/sardana2xls.py]: 7961083

File: sardana2xls/sardana2xls.py (hunk 1 above)
Comment (aureocarneiro) [reply sardana2xls/sardana2xls.py]: 7961083

**Pull request merged**